execution-environment/README.md

```diff
@@ -1,21 +1,31 @@
-## How to use execution environment
+# cloud.vmware_ops Execution Environment (EE)
 
-### Building
-To build the execution environment image:
+## Building
+
+You can build a container image capable of running the playbooks in this collection using `ansible-builder`.
+You can install it by running `pip install ansible-builder`
 
+To build the execution environment image:
 ```bash
-$ export USER=yourdockerusername
-$ ansible-builder build -t quay.io/$USER/vmware:mytest
+# change your working directory to this dir
+cd ./execution-environmnet
+
+# you may need to remove the context dir for a clean build
+rm -rf ./context
+
+ansible-builder build -t my_image_name:my_tag
 ```
 
-This will create container image `quay.io/$USER/vmware:mytest`. You can upload it to quay and use from AAP.
+This will create container image `my_image_name:my_tag`.
+You can then upload it to your container image repository (quay, docker hub, etc) and use from AAP.
+
 
-### Local test
-To test the image locally using `ansible-runner` execute the image as follows:
+## Running Locally
 
+You can run the container image locally `ansible-runner`.
+You can install it by running `pip install ansible-runner`
+
+As an example, you can run the info playbook with no options set using the following command:
 ```bash
-$ export USER=yourdockerusername
-$ ansible-runner run --container-image quay.io/$USER/vmware:mytest runner --inventory localhost, -p cloud.vmware_ops.security
+ansible-runner run --container-image my_image_name:my_tag runner -p cloud.vmware_ops.info /tmp
```

Comment:
this command failed with an error: `ansible-runner: error: unrecognized arguments: /tmp`

Since it is written above that a playbook will be run, it might be better to use the following format:
`ansible-runner run --container-image my_image_name:my_tag runner -p <path_to_playbook>/<playbook_file>`

Reply:
I had the incorrect command. I updated it to `ansible-runner run --container-image my_image_name:my_tag -p cloud.vmware_ops.info /tmp`

I cant seem to get the path to the playbook to work with ansible-runner. Im not sure why, im not familiar enough with ansible-runner and this use case.

Someone mentioned in the recent hackathon we need to make sure all of our playbooks are can be called using the collection namespace format (like `cloud.vmware_ops.info`) so i think that is the preferred way to reference them

Reply:
> ansible-runner run --container-image my_image_name:my_tag -p cloud.vmware_ops.info /tmp

I think this action will call the role itself, but not playbook.

> I cant seem to get the path to the playbook to work with ansible-runner. Im not sure why, im not familiar enough with ansible-runner and this use case.

Me neither, I specified the full path to playbook instead of playbook name, in this case I succeeded to run playbook.

> Someone mentioned in the recent hackathon we need to make sure all of our playbooks are can be called using the collection namespace format (like cloud.vmware_ops.info) so i think that is the preferred way to reference them

But what playbook it should run?

Reply:
>  think this action will call the role itself, but not playbook.

According to the ansible-runner doc, the `-p` flag runs playbooks and the `-r` flag runs modules. I checked the output of the command and I can see the line the shows the name of the playbook being run so it is running the action `info.yml` playbook. Im not entirely sure how the fully qualified naming works for playbooks, especially if they are in nested folders. We have a meeting with the product team to talk about that in the next week or so.

I was able to get your example working, so Ive updated the documentation to included that